### PR TITLE
fix some more translatable strings

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
* Change "Transposition against concert/sounding pitch:" to just "Transposition:" and add a toolTip to it, also add accessibilty text for the 2 input fields next to it.
* Take care of removing obsolete strings once again.
* Change "Patch" to "Sound", see https://musescore.org/node/292204

### CODE REVIEW

File: build/travis/job_macos_lupdate/script.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```
Comment (mirabilos) [inline build/travis/job_macos_lupdate/script.sh]: acceptable, this file does not use any bashisms at the moment

File: mscore/mixertrackchannel.cpp [hunk 2]
```diff
@@ -161,13 +161,13 @@ void MixerTrackChannel::updateNameLabel()
                            "Channel: %3\n"
                            "Bank: %4\n"
                            "Program: %5\n"
-                           "Patch: %6")
+                           "Sound: %6")
```
Comment (Jojo-Schmitz) [inline mscore/mixertrackchannel.cpp]: Maybe better leave it "Patch:" here, as here it really is the technical term?

File: mscore/mixertrackpart.cpp [hunk 3]
```diff
@@ -177,12 +177,12 @@ void MixerTrackPart::updateNameLabel()
                                 "Primary Instrument: %2\n"
                                 "Bank: %3\n"
                                 "Program: %4\n"
-                                "Patch: %5")
+                                "Sound: %5")
```
Comment (Jojo-Schmitz) [inline mscore/mixertrackpart.cpp]: same here

File: mscore/mixertrackpart.cpp [hunk 4]
```diff
@@ -177,12 +177,12 @@ void MixerTrackPart::updateNameLabel()
...
                   .arg(part->partName(),
                        part->longName(),
                        QString::number(chan->bank()),
                        QString::number(chan->program()),
-                       mp ? mp->name : tr("~no patch~"));
+                       mp ? mp->name : tr("~no sound~"));
```
Comment (Jojo-Schmitz) [inline mscore/mixertrackpart.cpp]: and probably here as well

File: mscore/mixertrackchannel.cpp [hunk 5]
```diff
@@ -161,13 +161,13 @@ void MixerTrackChannel::updateNameLabel()
...
                   .arg(part->partName(),
                        instr->trackName(),
                        qApp->translate("InstrumentsXML", chan->name().toUtf8().data()),
                        QString::number(chan->bank()),
                        QString::number(chan->program()),
-                       mp ? mp->name : tr("~no patch~"));
+                       mp ? mp->name : tr("~no sound~"));
```
Comment (Jojo-Schmitz) [inline mscore/mixertrackchannel.cpp]: and probably here as well

File: mscore/mixerdetails.ui [hunk 6]
```diff
@@ -466,7 +466,7 @@
           </size>
          </property>
          <property name="toolTip">
-          <string>Patch voice</string>
+          <string>Sound name</string>
```
Comment (Jojo-Schmitz) [inline mscore/mixerdetails.ui]: the 'voice' part doesn't seem to make any sense, or what am I missing?

File: mscore/mixertrackchannel.cpp (hunk 2 above)
Comment (Jojo-Schmitz) [reply mscore/mixertrackchannel.cpp]: Then again this here is the toolTip shown at the bottom, when hovering over the name fule below the volume slider, in ths same dialog as the one where now "Sound:" is shown at the top, where the sound can get selected, better those terms should match.

**Comment by mirabilos** [comment]
Now you’re conflating multiple issues, I cannot review on the others…

File: build/travis/job_macos_lupdate/script.sh [hunk 7]
```diff
@@ -9,12 +9,15 @@ fi
 
 # Translation routines
 # update translation on transifex
+# remove obsolete strings
+OBSOLETE=-no-obsolete # '-noobsolete' in older QT versions
```
Comment (anatoly-os) [inline build/travis/job_macos_lupdate/script.sh]: We use 5.9.8 for macOS builds.

Comment (Jojo-Schmitz) [reply build/travis/job_macos_lupdate/script.sh]: Yes, that 'older' referres to 'much older, Qt 4.8 or some such

**Pull request merged**